segmentationextractors/nwbextractor/nwbsegmentationextractor.py

```diff
@@ -4,10 +4,11 @@
 from dateutil.tz import tzlocal
 import numpy as np
 import re
-from ..segmentationextractor import SegmentationExtractor
+import yaml
+from segmentationextractors.segmentationextractor import SegmentationExtractor
 from lazy_ops import DatasetView
 from hdmf.data_utils import DataChunkIterator
-# from suite2p.io.nwb import save_nwb
+from segmentationextractors.nwbwriter import SegmentationExtractor2NWBConverter
```

Comment:
NWB Conversion Tools should import SegmentationExtractor, not the other way around. Please remove this.